apps/frontend/src/app/periods/[id]/data-table.tsx

```diff
@@ -249,22 +259,28 @@ export function DataTable<TData, TValue>({
               </TableCell>
             </TableRow>
             <TableRow>
-              <TableCell colSpan={columns.length} className='pt-0'>
-                <div className='flex flex-row gap-4 w-full justify-center'>
-                  <Button onClick={() => table.firstPage()} disabled={!table.getCanPreviousPage()}>
-                    {'<<'}
-                  </Button>
-                  <Button onClick={() => table.previousPage()} disabled={!table.getCanPreviousPage()}>
-                    {'<'}
-                  </Button>
-                  <div className='flex flex-col justify-center min-w-5 items-center'>{pagination.pageIndex}</div>
-                  <Button onClick={() => table.nextPage()} disabled={!table.getCanNextPage()}>
-                    {'>'}
-                  </Button>
-                  <Button onClick={() => table.lastPage()} disabled={!table.getCanNextPage()}>
-                    {'>>'}
-                  </Button>
-                </div>
+              <TableCell colSpan={columns.length}>
+                <Pagination className='mb-2'>
+                  <PaginationContent>
+                    <PaginationItem onClick={() => table.firstPage()} area-disabled={!table.getCanPreviousPage()}>
+                      <PaginationFirst />
+                    </PaginationItem>
+                    <PaginationItem onClick={() => table.previousPage()} area-disabled={!table.getCanPreviousPage()}>
+                      <PaginationPrevious />
+                    </PaginationItem>
+                    <PaginationItem>
+                      <PaginationLink href='#' isActive>
+                        {pagination.pageIndex + 1}
+                      </PaginationLink>
+                    </PaginationItem>
+                    <PaginationItem onClick={() => table.nextPage()} area-disabled={!table.getCanNextPage()}>
+                      <PaginationNext />
+                    </PaginationItem>
+                    <PaginationItem onClick={() => table.lastPage()} area-disabled={!table.getCanNextPage()}>
+                      <PaginationLast />
+                    </PaginationItem>
```

Comment:
these are not disabled when there are just a few pages.
I'm pretty sure there is no `area-disabled` attribute, only `aria-disabled`, and even that only affects screen readers

Reply:
valid point